packages/fxa-settings/src/styles/UnitRow.scss

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+.unit-row {
+  @apply flex flex-wrap px-4 pt-7 pb-8;
+
+  &-header {
+    @apply w-full mb-1;
+  }
+
+  &-header-value {
+    @apply w-full;
+  }
+
+  &-cta-container {
```

Comment:
Because we'll be putting more than standard CTA buttons, such as icon buttons, and in theory we could put other content here down the road I wonder if this could be more generic, such as `&-actions`, or `&-extra` (less enthused about this).

Reply:
Good point, I'm fine with `&-actions` due do that "delete" button we'll add later. (I guess that _could_ be looked at as a call to action, but eh)